### DISCUSSION TITLE
whisper Vue_Devtools.mp4 command does not work for me

### DISCUSSION THREAD

**Discussion by mwangaben** [openai/whisper#1179] [Show and tell]
Hi there i am on MacOs Mojave  with Python3.10 , i  can seems to get the desired result . I got this when run

<img width="1432" alt="Screenshot 2023-04-01 at 07 06 06" src="https://user-images.githubusercontent.com/17113999/229265117-9fd3150f-7867-49c2-bf03-1094c44d0793.png">

   `whisper Vue_Devtools.mp4`

Please assist

**Comment by glangford** [comment]
There is a second step required when you install Python on Mac OS. Go to Applications/Python 3.10 and run Install Certificates.command.
